packages/@atjson/source-ckeditor/test/source-classic-build-test.ts

```diff
@@ -216,4 +216,90 @@ describe("@atjson/source-ckeditor classic build", () => {
       }
     ]);
   });
+  test("update children start/end positions to enforce nesting hierarchy", () => {
+    editor.setData(
+      "<strong><em>Bold and italic</em></strong> <strong><a href='https://www.condenast.com'>bold link</a> just bold</strong>"
+    );
+    let doc = CKEditorSource.fromRaw(editor.model);
+
+    expect(doc.content).toBe("\uFFFC\uFFFCBold and italic bold link just bold\uFFFC\uFFFC");
+    expect(doc.annotations.sort(compareAnnotations)).toMatchObject([
```

Comment:
Can we use `sort` without arguments here?

Reply:
If we update the sort, yes, but the default sort without arguments is so unintuitive that it makes it harder to write these tests

Reply:
I don't think the default sort does anything at all since it's working on an array of annotation objects rather than strings. Eg it doesn't go into each annotation and does a string comparison on each annotation type by default

Reply:
Oh yeah, 😭